src/main/java/com/uid2/admin/audit/QLDBAuditWriter.java

```diff
@@ -17,6 +17,7 @@
 public class QLDBAuditWriter implements AuditWriter{
     private static final IonSystem ionSys = IonSystemBuilder.standard().build();
     private static final Logger logger = LoggerFactory.getLogger(QLDBAuditWriter.class);
+    private static final Logger auditLogger = LoggerFactory.getLogger("com.uid2.admin.audit");
```

Comment:
do you need booth loggers?

Reply:
Yes, one logger writes to the separate loki audit log, the other is for error writing / standard loki log